### DISCUSSION TITLE
TODO verificar porque 'svg' é com letras minúsculas

### DISCUSSION THREAD

**Discussion by WinterSilence** [FilipePS/Traduzir-paginas-web#401] [Show and tell]
https://github.com/FilipePS/Traduzir-paginas-web/blob/f61f591d7c6f73d36b44625659d5ceee6a274730/contentScript/pageTranslator.js#L54

- For DOM trees which represent HTML documents, the returned tag name is always in the canonical upper-case form. For example, `tagName` called on a `<div>` element returns `DIV`.
- The tag names of elements in an XML DOM tree are returned in the same case in which they're written in the [original XML file](http://www.w3.org/2000/svg). If an XML document includes a tag `<SomeTag>`, then the `tagName` property's value is `SomeTag`.

**Comment by FilipePS** [comment]
`svg` is always lowercase when returned by `tagName`, regardless of how it was originally written.
So I think this explanation is not valid.

**Comment by WinterSilence** [comment]
@FilipePS [original XML file](http://www.w3.org/2000/svg) i.e SVG specification, not your files (:
